docs/how-to/data-plane-installation.md

```diff
@@ -0,0 +1,74 @@
+# Installing Kuadrant data-plane into an existing OCM Managed Cluster
+
+## Introduction
+This walkthrough will show you how to install and setup the Kuadrant Operator into an [OCM](https://open-cluster-management.io/) [Managed Cluster](https://open-cluster-management.io/concepts/managedcluster/).
+
+Steps:
+- OCM should be installed and you should have an existing OCM hub cluster (we should cover the specifics of that setup in the cp install guide which this should link to) with the control plane components installed (IE follow the CP guide first)
```

Comment:
Should this be linking out to an OCM install guide? Something along the lines of what's outlined [here] (https://github.com/Kuadrant/multicluster-gateway-controller/blob/main/docs/how-to/ocm-control-plane-walkthrough.md) via the quickstart script?

Reply:
Perhaps work to add a CP install guide is already underway, if it is feel free to ignore the above comment

Reply:
A link to a guide would help, although I guess we'd link to upstream OCM - this guide is moreso about installing data-plane pieces onto an existing OCM, so unlikely they would've used our guides to install OCM originally. Although it wouldn't hurt to mention using our quickstart guide would give them a good way to do initial tyre kicking.

Reply:
OK - added both!